notebooks/04_model_select_and_optimize/tuning_spark_als.ipynb

```diff
@@ -37,16 +37,16 @@
   },
```

Comment:
<p>this is a good idea, you could add the ratio to the parameters section and we could even crank that down further for unit test vs. others.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/864/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>